Changelog.md

```diff
@@ -15,7 +15,7 @@
 ### Bug Fixes
 
 ### Misc Changes
-
+- [#594]: Add documentation for using tagged enums with Serde.
```

Comment:
Link should be actually added:
```suggestion
- [#594]: Add documentation for using tagged enums with Serde.

[#594]: https://github.com/tafia/quick-xml/pull/594

```

Also check `cargo fmt` -- it seems that there are some trailing spaces!